packages/components/buttons/link-button/src/link-button.tsx

```diff
@@ -11,6 +12,40 @@ import {
 import Inline from '@commercetools-uikit/spacings-inline';
 import Text from '@commercetools-uikit/text';
 
+export type TLinkButtonProps = {
```

Comment:
Converted to TS so that the `buttons` preset can re-export all type declarations.